src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionOption/ColorPicker/ColorPicker.tsx

```diff
@@ -10,29 +10,39 @@ export const ColorPicker = <T extends FieldValues>({
   anchorEl,
   handlePopoverClose,
 }: ColorPickerProps<T>) => {
+  const { setValue } = useFormContext();
+
   const colorPickerVisible = Boolean(anchorEl);
 
   return (
     <Controller
       name={name}
       control={control}
-      render={({ field: { onChange, value } }) => (
-        <Popover
-          open={colorPickerVisible}
-          onClose={handlePopoverClose}
-          anchorEl={anchorEl}
-          anchorOrigin={{
-            vertical: 'bottom',
-            horizontal: 'center',
-          }}
-          transformOrigin={{
-            vertical: 'top',
-            horizontal: 'center',
-          }}
-        >
-          <ChromePicker disableAlpha={true} color={value} onChangeComplete={onChange} />
-        </Popover>
-      )}
+      render={({ field: { onChange, value } }) => {
+        const handleChange: ColorChangeHandler = (...args) => {
+          onChange(...args);
+
+          setValue('palette', '');
```

Comment:
Should not we use 'palette' as a paletteName property for the component?

Reply:
done